builder/azure/arm/builder_acc_test.go

```diff
@@ -26,22 +29,93 @@ import (
 	"bytes"
 	"context"
 	_ "embed"
-	"errors"
 	"fmt"
 	"io/ioutil"
 	"os"
 	"os/exec"
-	"strings"
 	"testing"
+	"time"
 
-	"github.com/Azure/go-autorest/autorest"
-	"github.com/Azure/go-autorest/autorest/azure/auth"
+	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2021-11-01/compute"
 	"github.com/hashicorp/packer-plugin-sdk/acctest"
 	packersdk "github.com/hashicorp/packer-plugin-sdk/packer"
+	"github.com/hashicorp/packer-plugin-sdk/retry"
 )
 
 const DeviceLoginAcceptanceTest = "DEVICELOGIN_TEST"
 
+func TestBuilderAcc_ARM64SpecializedLinuxSIG(t *testing.T) {
```

Comment:
For ease of testing I suggest renaming these test a to include SIG or SharedImageGallery
```suggestion
func TestBuilderAcc_SharedImageGallery_ARM64SpecializedLinuxSIG(t *testing.T) {
```

This way we can run `PACKER_ACC=1 go test ./... -run= 'TestBuilderAcc_SharedImageGallery'` or just  `PACKER_ACC=1 go test ./... -run= 'SharedImageGallery'`